File: Service/Admin/RMA.Service.Admin.MasterDataManager.Contracts/Entities/FollowUp.cs
using RMA.Common.Entities;

using System;

namespace RMA.Service.Admin.MasterDataManager.Contracts.Entities
{
    public class FollowUp : AuditDetails
    {
        public string Name { get; set; }
        public int ItemId { get; set; }
        public string ItemType { get; set; }
        public TimeSpan AlertTime => AlertDate.TimeOfDay;
        public string Description { get; set; }
        public DateTime AlertDate { get; set; }
        public string Email { get; set; }
        public bool AlertSent { get; set; }
        public string Reference { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/clientcare/src/app/policy-manager/shared/entities/follow-up.ts
import { BaseClass } from 'projects/shared-models-lib/src/lib/common/base-class';

export class FollowUp extends BaseClass {
    recipientTypeId: number;
    itemId: number;
    itemType: string;
    userId: number;
    policyId: number;
    name: string;
    email: string;
    description: string;
    alertDate: Date;
    alertTime: Date;
    minutes: number;
    hours: number;
    reference: string;
}
